packages/client/src/variables.scss

```
@@ -0,0 +1,21 @@
+/* Colors */
+$background-color: rgb(250, 248, 239);
+$font-color: rgb(119, 110, 101);
+$light-brown: rgb(187, 173, 160);
+$dark-brown: rgb(143, 122, 102);
```

Насколько я знаю, лучше бы такие переменные именовать как:
```suggestion
$color-primary: rgb(187, 173, 160);
$color-secondary: rgb(143, 122, 102);
```
Поскольку в дальнейшем при смене этого цвета с коричневого, на любой другой, то смысл названия потеряется

---

Понимаю твою идею, но пока не могу определить, какой из этих цветов primary/secondary. Светлый используется однократно для бекграунда страниц, а темный -  также, однократно, для кнопок.
Можно оставить этот вопрос до следующих реквестов, когда станет чуть понятнее с дальнейшим использованием этих цветов.